src/script/E2EIdentity/E2EIdentity.ts

```diff
@@ -142,7 +133,7 @@ class E2EIHandler {
       let oAuthIdToken: string | undefined;
 
       // If the enrollment is in progress, we need to get the id token from the oidc service, since oauth should have already been completed
-      if (this.coreE2EIService.isEnrollmentInProgress()) {
+      if (this.core.service?.e2eIdentity?.isEnrollmentInProgress()) {
```

Comment:
Please revert this change. It is better to have a check of it being defined like we had before